src/EnergyPlus/HeatingCoils.cc

```diff
@@ -2842,18 +2793,34 @@ namespace HeatingCoils {
                         state.dataHeatBal->HeatReclaimRefrigCondenser(SourceID).AvailCapacity * Effic -
                         state.dataHeatBal->HeatReclaimRefrigCondenser(SourceID).WaterHeatingDesuperheaterReclaimedHeatTotal;
                 }
-            } else if (HeatingCoil(CoilNum).ReclaimHeatingSource == HeatObjTypes::COIL_DX_COOLING ||
-                       HeatingCoil(CoilNum).ReclaimHeatingSource == HeatObjTypes::COIL_DX_MULTISPEED ||
-                       HeatingCoil(CoilNum).ReclaimHeatingSource == HeatObjTypes::COIL_DX_MULTIMODE) {
+                break;
+            case HeatObjTypes::COIL_DX_COOLING:
+            case HeatObjTypes::COIL_DX_MULTISPEED:
+            case HeatObjTypes::COIL_DX_MULTIMODE:
                 HeatingCoil(CoilNum).RTF = state.dataDXCoils->DXCoil(SourceID).CoolingCoilRuntimeFraction;
                 HeatingCoil(CoilNum).NominalCapacity = state.dataHeatBal->HeatReclaimDXCoil(SourceID).AvailCapacity * Effic -
                                                        state.dataHeatBal->HeatReclaimDXCoil(SourceID).WaterHeatingDesuperheaterReclaimedHeatTotal;
-            } else if (HeatingCoil(CoilNum).ReclaimHeatingSource == HeatObjTypes::COIL_DX_VARIABLE_COOLING) {
+                break;
+            case HeatObjTypes::COIL_DX_VARIABLE_COOLING:
                 // condenser heat rejection
                 HeatingCoil(CoilNum).RTF = state.dataVariableSpeedCoils->VarSpeedCoil(SourceID).RunFrac;
                 HeatingCoil(CoilNum).NominalCapacity = state.dataHeatBal->HeatReclaimVS_DXCoil(SourceID).AvailCapacity * Effic -
                                                        state.dataHeatBal->HeatReclaimVS_DXCoil(SourceID).WaterHeatingDesuperheaterReclaimedHeatTotal;
+                break;
+            case HeatObjTypes::COIL_COOLING_DX_NEW:
+                // get RTF and NominalCapacity from Coil:CoolingDX
+                {
+                    auto &thisCoolingCoil = state.dataCoilCooingDX->coilCoolingDXs[SourceID];
+                    HeatingCoil(CoilNum).RTF = thisCoolingCoil.runTimeFraction;
```

Comment:
I see this is how is has been done in the past, but this should be PLR since PLR relates to runtime and RTF relates to power. This is a follow up issue.

Reply:
Since this->reclaimHeat.AvailCapacity = this->totalCoolingEnergyRate + this->elecCoolingPower, which is air-side energy (PLR) + electricity (RTF), there is a bit of a problem with the portion of the time step each of these represent. Again, for another time.

Reply:
Now I think RTF is correct in this context.